### PR TITLE
ninja review

### PR DESCRIPTION
**Pull request by mandarbapaye**

### CODE REVIEW

**Comment by mandar-masterclass** [comment]
not good

File: pull-1.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 This is done by mandar-masterclass-again.
 pull-2
-pull-3
```
Comment (mandarbapaye) [inline pull-1.txt:3]: !fix

Comment (mandarbapaye) [inline pull-1.txt:3]: !resolve

Comment (mandarbapaye) [inline pull-1.txt:3]: !completed

Comment (mandarbapaye) [inline pull-1.txt:3]: !fix

Comment (mandar-masterclass) [inline pull-1.txt:3]: I think this is fine. !completed

File: pull-1.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
\ No newline at end of file
+pull-4
```
Comment (mandar-masterclass) [inline pull-1.txt:3]: !fix

Comment (mandar-masterclass) [inline pull-1.txt:3]: !completed

**Comment by mandarbapaye** [comment]
This is from chrome - github directly

File: pull-1.txt [hunk 3]
```diff
@@ -1,3 +1,4 @@
 This is done by mandar-masterclass-again.
 pull-2
-pull-3
\ No newline at end of file
```
Comment (mandar-masterclass) [inline pull-1.txt:3]: pull-4 should be pull-5

**Pull request closed without merging**